### ISSUE TITLE
Running as background service

### ISSUE THREAD

**Issue by autoantwort** [RobertKrajewski/Sync-my-L2P#150]
It would be nice if the application can run in the background only showing a [SystemTrayIcon](https://doc.qt.io/qt-5/qsystemtrayicon.html) like [Telegram Desktop](https://desktop.telegram.org/) (also written in Qt) and syncing every hour.

**Comment by RobertKrajewski** [comment]
Yeah, I also had this idea many years ago and implemented the current version of the Tray Icon. However, the current (bad) design of the necessary components prevented the implementation of this feature without redesigning some components. Further, #45 might still persist.
I wont implement this feature, but am open to contributions
